test_accounts/benefits_test_accounts.md

```diff
@@ -81,22 +83,31 @@ These test users have both Claims data and Health data and may be used in both t
 | :--- | :--- | :--- | :--- | :--- | :--- |
 |va.api.user+idme.101@gmail.com|Sheba703|Harris789|F|1926-01-08|111111111|
 
-## Representative
+## Representatives
 
-These test users have Power of Attorney (POA) for specific Veterans and can be used to test the [Benefits Claims API](https://developer.va.gov/explore/benefits/docs/claims?version=current). During the authentication process, we look up the Office of General Counsel (OGC) POA code associated with the Veteran and validate that the Representative has a matching POA code.
+There are two test users, Tamara Ellis and John Doe, who are accredited and can be assigned Power of Attorney (POA) for specific Veterans. They can be used to test the [Benefits Claims API](https://developer.va.gov/explore/benefits/docs/claims?version=current). During the authentication process, we look up the Office of General Counsel (OGC) POA code associated with the Veteran and validate that the Representative has a matching POA code.
 
 **Both Veterans and non-Veterans can be Representatives.**
 
-Test user 007 (John Doe) is a non-Veteran that can be added as a POA for any of the above Veterans (if not already). If test user 007 attempts to read or write data for a Veteran test user they do not have POA for, the server will return a 401 Unauthorized error.
-
-Also, any of the above Veterans can also be Representatives for another Veteran. In addition to accessing the VA data for those the Veteran represents, he/she can also access his/her own personal VA data. If the Representative attempts to read or write data for a Veteran test user they do not have POA for, the server will return a 401 Unauthorized error.
+Tamara Ellis, test user 001, is a Veteran. John Doe, test user 007, is not. Both can assigned POA for any of the above Veterans, if not already. If the Representative attempts to read or write data for a Veteran test user they do not have POA for, the server will return a `401 Unauthorized` error.
```

Comment:
`Both can assigned POA` -> `Both can be assigned POA`

Reply:
[Done](https://github.com/department-of-veterans-affairs/vets-api-clients/pull/309/commits/66467bdcf6ab8caaa271e81bb13d37cc84000ac9)